docs/helpers/ObjectSerializer.md

```diff
@@ -0,0 +1,81 @@
+title: Object Serializer
+author: simop
+description: IObjectSerializer is an interface that you can implement to provide a serializer of your choice to ObjectStorageHelper.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, serialization
+dev_langs:
+  - csharp
+  - vb
+---
+
+# ObjectSerializer
+
+You should implement IObjectSerializer when you need to write data using this toolkit's helpers with a custom serializer. If you don't, a default Json serializer will be used otherwise.

```

Comment:
```suggestion
You should implement IObjectSerializer when you need to write data using this toolkit's helpers with a custom serializer. If you don't, a default JSON serializer will be used otherwise.

```